js/components/portal-dashboard/all-responses-popup/student-responses-popup.tsx

```diff
@@ -24,33 +25,37 @@ interface IProps {
 }
 export class StudentResponsePopup extends React.PureComponent<IProps> {
   render() {
-    const { anonymous, students, isAnonymous, studentCount, setAnonymous, setStudentFilter, trackEvent,
-      currentQuestion, questions, sortedQuestionIds, toggleCurrentQuestion, setCurrentActivity } = this.props;
+    const { anonymous, currentQuestion, isAnonymous, isHidden, questions, setCurrentActivity, setAnonymous,
+            setStudentFilter, sortedQuestionIds, studentCount, students, toggleCurrentQuestion, trackEvent } = this.props;
     return (
-      <div className={css.popup} data-cy="all-responses-popup-view">
+      <div className={`${css.popup} ${isHidden ? css.hidden : ""}`} data-cy="all-responses-popup-view">
         <PopupHeader handleCloseAllResponsesPopup={this.props.handleCloseAllResponsesPopup} />
         <div className={css.tableHeader}>
-          <PopupClassNav
-            anonymous={anonymous}
-            studentCount={studentCount}
-            setAnonymous={setAnonymous}
-            setStudentFilter={setStudentFilter}
-            trackEvent={trackEvent} />
-          <div className={css.questionArea} data-cy="questionArea">
+        <PopupClassNav
+          anonymous={anonymous}
+          studentCount={studentCount}
+          setAnonymous={setAnonymous}
+          setStudentFilter={setStudentFilter}
+          trackEvent={trackEvent} />
+        <div className={css.questionArea} data-cy="questionArea">
+          { currentQuestion &&
```

Comment:
Indent one less?

Reply:
This one should be indented as it is inside of the div not adjacent to it.